hscommon/build.py

```diff
@@ -557,7 +557,7 @@ def fix_qt_resource_file(path):
     with open(path, "rb") as fp:
         contents = fp.read()
     lines = contents.split(b"\n")
-    lines = [l for l in lines if not l.startswith(b"#")]
+    lines = [lin for lin in lines if not lin.startswith(b"#")]
```

Comment:
Minor request, would you mind changing `lin` to `line`.  N-1 abbreviations irritate me.